content/_admin/ip-whitelisting.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Egress IP Addresses & IP Whitelisting
+layout: article
+section: elastic.io Specific Settings
+description: "Document lists public IP addresses of our clusters to whitelist if required."
+order: 1
+category: support
+---
+
+**Summary**
+  * *Traffic from the {{site.data.tenant.name}} platform comes from fixed IP addresses*
+  * *The fixed IP addresses are listed on this page*
+  * *{{site.data.tenant.companyName}} commits to supporting the ability to allowing integrators to add {{site.data.tenant.name}} IP addresses to their whitelists*
+<hr />
+
+IP connections that originate from a step within a flow within the {{site.data.tenant.name}} platform will be randomly routed through one of the IP addresses listed below. If you have enabled IP whitelisting on your server, all IP addresses listed below should be added to the IP whitelist of that server.
+
+The IP addresses listed below are common to all {{site.data.tenant.name}} users. If you would like to have egress IP traffic routed through an IP address that is exclusive to you, then the [VPN agents](/guides/vpn-agent) feature can be used to achieve this. 
+
+In the future, {{site.data.tenant.companyName}} may need to add or remove IP addresses from the lists below. In this case, {{site.data.tenant.companyName}} will proactively inform you of these changes.
+
+## IP Whitelist for app.elastic.io
```

Comment:
'IP Whitelist' should be 'IP List' or 'IP Addresses'. If a user simply wants to know the address for debugging or for whitlisting is not known to the auther of the page and should not be assumed. (Same applies to the next section heading).